documentation/modules/snip_secure_boot_issue.adoc

```diff
@@ -0,0 +1,7 @@
+:_content-type: SNIPPET
+
+.VMs with Secure Boot enabled might not be migrated automatically
+
+Virtual machines (VMs) with Secure Boot enabled currently might not be migrated automatically. This is because Secure Boot, a security standard developed by members of the PC industry to ensure that a device boots using only software that is trusted by the Original Equipment Manufacturer, would prevent the VMs from booting on the destination provider. 
```

Comment:
```suggestion
Virtual machines (VMs) with Secure Boot enabled currently might not be migrated automatically. This is because Secure Boot, a security standard developed by members of the PC industry to ensure that a device boots using only software that is trusted by the Original Equipment Manufacturer (OEM), would prevent the VMs from booting on the destination provider. 
```